app/models/opentofu_worker.rb

```diff
@@ -10,7 +10,8 @@ class OpentofuWorker < MiqWorker
     %i[workers worker_base opentofu_worker]
   ]
 
-  OPENTOFU_RUNTIME_DIR = "/var/lib/manageiq/opentofu-runner".freeze
+  MIQ_CONTAINERS_ROOT_DIR = "/var/lib/manageiq/containers/storage".freeze
```

Comment:
How will this affect local dev env (and I guess test env)?  I'm wondering if we should gate this on Rails.env.production?

Reply:
https://github.com/ManageIQ/manageiq-providers-embedded_terraform?tab=readme-ov-file#running-opentofu-runner-locally is how you run this worker locally.

Because this doesn't go through run_single_worker this has a different way to run it for development at least at present.

Reply:
Ok yeah good point.